Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/datamgr/actions/FindDataTypesContainingAction.java

```diff
@@ -238,8 +238,6 @@ else if (dt instanceof FunctionDefinition) {
 		}
 		else if (dt instanceof BuiltInDataType) {
 		}
-		else if (dt instanceof MissingBuiltInDataType) {
-		}
 		else if (dt.equals(DataType.DEFAULT)) {
 
 		}
```

Comment:
These could be combined. Also I don't see why MissingBuiltInDataType is being removed.

Reply:
See https://github.com/NationalSecurityAgency/ghidra/pull/1755#discussion_r407325035